Gemfile

```diff
@@ -80,7 +80,7 @@ gem 'daemons'
 # (see https://www.openproject.org/work_packages/3029)
 gem 'rack-protection', :git => "https://github.com/finnlabs/rack-protection.git", :ref => '5a7d1bd'
 
-gem 'syck', :platforms => [:ruby_20, :mingw_20, :ruby_21, :mingw_21], :require => false
+gem 'syck', :platforms => [:mri, :mingw, :x64_mingw], :require => false
```

Comment:
Why are you replacing the `ruby` platform with `mri`? As far as I can tell this will exclude rubinius.

I am also not quite sure which platform are left to exclude the `syck` gem. I.e. is the platform key needed at all?